### DISCUSSION TITLE
gpt4-o内容多了之后无法显示

### DISCUSSION THREAD

**Discussion by forrany** [BurnHair/docs#102] [技术支持]
如题，在 lobe-chat 中，当 gpt-4o 的内容多了之后，就会无法显示

如下图
<img width="1802" alt="image" src="https://github.com/BurnHair/docs/assets/29779326/620b9620-a8e4-4f90-81a1-759588c3bee5">

<img width="1236" alt="image" src="https://github.com/BurnHair/docs/assets/29779326/462cb356-849e-4ca1-b6db-99597c4d8855">

**Comment by BennyThink** [comment]
试试更新 lone chat到最新版

**Reply by jkjoy** [reply]
超过字符没写完不要紧

回复 `continue` 会继续续写
